.github/pull_request_template.md

```diff
@@ -6,12 +6,13 @@ Please write a few sentences describing the overall goals of the pull request's
 
 ## PR checklist
 
-Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")
+Please add/remove strikethrough ~~ as appropriate, and add any relevant link/s to the end of the line.
```

Comment:
Not clear to me: if I want to add a justification, where should I add it? Before of after the link?

Reply:
Originally the [proposal was](https://github.com/Mbed-TLS/mbedtls/pull/9008#issuecomment-2084739951) to have a comments section after the special code.

That way it would be easier to parse as a block of text without affecting the template.

e.g. 

~~~~~~
- [ ] **dev PR** provided, ~~not required~~ (Justification/Comments). Link --> #XYZ
~~~~~~

But I removed it since the comment's suggested that we should not increase the text of the checkboxes.

I am happy either way.